token-price-oracle/Makefile

```
@@ -0,0 +1,39 @@
+.PHONY: build test lint clean run docker-build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)